lib/services/network_helper.dart

```diff
@@ -1,17 +1,18 @@
+import 'dart:convert';
+
 import 'package:http/http.dart' as http;
 
 class NetworkHelper {
   NetworkHelper(this.url);
   final String url;
 
-  Future<String> getData() async {
+  Future<Map<String, dynamic>> getData() async {
     http.Response response = await http.get(url);
 
     if (response.statusCode == 200) {
-      return response.body;
+      return json.decode(response.body);
     } else {
-      print(response.statusCode);
-      return null;
+      throw Exception('Failed to load api.');
```

Comment:
Nice 💯, now we only decode once rather than decoding every time we call this method.